api/v1alpha1/datadogagent_types.go

```diff
@@ -295,6 +295,13 @@ type APMSpec struct {
 	// +listMapKey=name
 	Env []corev1.EnvVar `json:"env,omitempty"`
 
+	// Specify additional volume mounts in the Process Agent container
```

Comment:
```suggestion
	// Specify additional volume mounts in the APM/Trace Agent container
```

Reply:
There's three separate ones, but indeed the comment is wrong for this one. Nice spot, fixed!